docs/docs/genai/prompt-version-mgmt/prompt-registry/index.mdx

```diff
@@ -483,7 +483,8 @@ A: You can delete a prompt version using the MLflow UI or Python API:
 import mlflow
 
 # Delete a prompt version
-mlflow.delete_prompt("summarization-prompt", version=2)
+client = mlflow.MlflowClient()
+client.delete_prompt_version("summarization-prompt", version=2)
```

Comment:
`mlflow.delete_prompt` has been removed. Use the client `delete_prompt_version`.